python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_evaluation_info.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+import itertools
+from typing import Any, Optional, Sequence
+
+from dagster._core.asset_graph_view.asset_graph_view import AssetGraphView, TemporalContext
+from dagster._core.definitions.asset_key import AssetKey
+from dagster._core.definitions.asset_subset import AssetSubset
+from dagster._core.definitions.declarative_scheduling.serialized_objects import (
+    AssetConditionEvaluation,
+    AssetConditionEvaluationState,
+    AssetSubsetWithMetadata,
+)
+from dagster._model import DagsterModel
+
+
+class SchedulingEvaluationNode(DagsterModel):
```

Comment:
Hmm why name this `SchedulingEvaluationNode`? What does this represent precisely?

Reply:
Per discussion these objects represent the results of evaluation and retain the tree structure for lineage, etc. One of the classes should change names to indicate that.

Reply:
resolving: renamed to SchedulingEvaluationResultNode / updated docstring (rename happens one PR up due to a graphite mishap)

Reply:
got i.t